packages/bp/src/sdk/botpress.d.ts

```diff
@@ -443,7 +443,7 @@ declare module 'botpress/sdk' {
   }
 
   export namespace IO {
-    export type EventDirection = 'incoming' | 'outgoing'
+    export type EventDirection = 'incoming' | 'outgoing' | 'internal'
```

Comment:
do you really need to add a direction ? could you do without ? 
it'll break typings pretty much everywhere we check EventDirection. I remember there are typeguards everywhere they won't be typesafe anymore 😢